workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -247,42 +247,53 @@ def diurnal_component_plot(
 
 # Routines for plotting the "diagnostics"
 @timeseries_plot_manager.register
-def rms_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def rms_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d(diagnostics, varfilter="rms_global")
 
 
 @timeseries_plot_manager.register
-def spatial_mean_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def spatial_mean_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d_with_region_bar(diagnostics, varfilter="spatial_mean")
 
 
 @timeseries_plot_manager.register
-def spatial_minmax_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def spatial_minmax_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d_min_max_with_region_bar(
         diagnostics, varfilter_min="spatial_min", varfilter_max="spatial_max"
     )
 
 
 @timeseries_plot_manager.register
-def custom_timeseries_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def custom_timeseries_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d(diagnostics, varfilter="timeseries")
 
 
 @zonal_mean_plot_manager.register
-def zonal_mean_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def zonal_mean_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d(diagnostics, varfilter="zonal_and_time_mean")
 
 
 @hovmoller_plot_manager.register
-def zonal_mean_hovmoller_plots(diagnostics: Iterable[xr.Dataset]) -> RawHTML:
+def zonal_mean_hovmoller_plots(diagnostics: RunDiagnostics) -> RawHTML:
     return plot_2d_matplotlib(diagnostics, "zonal_mean_value", ["time", "latitude"])
 
 
 @hovmoller_plot_manager.register
-def zonal_mean_hovmoller_bias_plots(diagnostics: Iterable[xr.Dataset]) -> RawHTML:
+def zonal_mean_hovmoller_bias_plots(diagnostics: RunDiagnostics) -> RawHTML:
     return plot_2d_matplotlib(diagnostics, "zonal_mean_bias", ["time", "latitude"])
 
 
+@hovmoller_plot_manager.register
+def zonal_mean_short_term_bias_plots(diagnostics: RunDiagnostics) -> RawHTML:
+    return plot_2d_matplotlib(
+        diagnostics.trim_duration(np.timedelta64(3, "D")),
```

Comment:
Nit: If `duration` is already required to be a `np.timedelta64` at this point, move the function call inside `trim_duration` and just pass the `*args` to it.

Reply:
I think it's more straightforward to type-hint a `np.timedelta64` duration than to type-hint the args that get passed to `np.timedelta64` (and in principle the duration could be generated some other way, like taking the difference between two times).